onboarding/php.md

```diff
@@ -1,61 +1,86 @@
-## Installation
+### Installation
 
-### General
-
-Download <a href="https://raw.githubusercontent.com/rollbar/rollbar-php/v0.18.2/src/rollbar.php" target="_blank" rel="noopener">rollbar.php</a> and <a href="https://raw.githubusercontent.com/rollbar/rollbar-php/v0.18.2/src/Level.php" target="_blank" rel="noopener">Level.php</a>
-and put them together somewhere you can access.
-
-### If Using Composer
+## Installation with Composer (recommended)
 
 Add `rollbar/rollbar` to your `composer.json`:
 
 ```json
 {
     "require": {
-        "rollbar/rollbar": "~0.18.2"
-    }
-}
-```
-
-### Upcoming Release
-
-If you'd like to run the next release of rollbar-php, you can use it via composer by adding the following to
-your `composer.json`
-
-```json
-{
-    "require": {
-        "rollbar/rollbar": "~1.0.0-beta"
+        "rollbar/rollbar": "~1.0"
     }
 }
 ```
 
-## Send a Message to Rollbar
+## Manual Installation
+Even if you're not using composer for your project, you still need the composer package to install rollbar-php dependencies.
 
-Add the following code to one of your PHP pages:
+1. If you don't have composer yet, follow these instructions to get the package: [install composer](https://getcomposer.org/doc/00-intro.md).
+2. Clone git repository rollbar/rollbar-php into a your external libraries path: git clone https://github.com/rollbar/rollbar-php
+3. Install rollbar-php dependencies: cd rollbar-php && composer install && cd ..
+4. Require rollbar-php in your PHP scripts: require_once YOUR_LIBS_PATH . '/rollbar-php/vendor/autoload.php';
 
+### Send a Test Message
+Add the following code at your application's entry point:
 ```php
 <?php
-// installs global error and exception handlers
-Rollbar::init(array('access_token' => '{{ server_access_token }}'));
-
-// Message at level 'info'
-Rollbar::report_message('testing 123', 'info');
+use \Rollbar\Rollbar;
+
+// Installs global error and exception handlers
+$config = array(
+    // required
+    // replace 'POST_SERVER_ITEM_ACCESS_TOKEN' with your project's post_server_item
+    'access_token' => 'POST_SERVER_ITEM_ACCESS_TOKEN',
```

Comment:
In the onboarding flow, we can fill in the actual access token - see the old line 40, above.